www-client/vivaldi-snapshot/vivaldi-snapshot-6.3.3119.4.ebuild

```diff
@@ -101,10 +101,11 @@ fi
 KEYWORDS="-* ~amd64 ~arm ~arm64"
 VIVALDI_BASE_URI="https://downloads.vivaldi.com/${VIVALDI_PN#vivaldi-}/${VIVALDI_PN}_${PV%_p*}-${DEB_REV}_"
 
-RE="\bamd64\b"; [[ ${KEYWORDS} =~ ${RE} ]] && SRC_URI+=" amd64? ( ${VIVALDI_BASE_URI}amd64.deb )"
-RE="\barm\b"; [[ ${KEYWORDS} =~ ${RE} ]] && SRC_URI+=" arm? ( ${VIVALDI_BASE_URI}armhf.deb )"
-RE="\barm64\b"; [[ ${KEYWORDS} =~ ${RE} ]] && SRC_URI+=" arm64? ( ${VIVALDI_BASE_URI}arm64.deb )"
-RE="\bx86\b"; [[ ${KEYWORDS} =~ ${RE} ]] && SRC_URI+=" x86? ( ${VIVALDI_BASE_URI}i386.deb )"
+SRC_URI="
```

Comment:
This isn't quite the same, as the idea is to check if it's in KEYWORDS, and allow dropping from KEYWORDS if the packageh as no binary available right now.

But I have no idea how often that happens or why this bit is there to begin with..

Reply:
I think it is more readable (and faster in global scope) to just list them close one to another. As a result I think the new variant is better.

Reply:
yeah, I prefer arthur's new code, way better